frontend/src/metabase/admin/tasks/containers/BugReportApp.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import { Box } from "grid-styled";
```

Comment:
I'm not very strict about this (and this is small and already pretty close), but here's my preferred import order: https://github.com/metabase/metabase/wiki/Frontend:-Style-Guide#additional-javascript-conventions

The only difference would be moving `ttag` below `react`.